content/millennium/r4/documents/document-reference.md

```diff
@@ -191,7 +191,8 @@ Update an existing document reference:
 <%= authorization_types(provider: true, patient: false, system: true) %>
 
 _Implementation Notes_
-
+* Any field which is missing will be interpreted as nulling out or removing data from the resource. See [FHIR<sup>®</sup> Update] for additional details about update operations.
+* Both read and write scopes are required.
 * Any field which is missing will be interpreted as nulling out or removing data from the resource. Refer to the HL7<sup>®</sup> FHIR<sup>®</sup> [Update documentation] for additional details about update operations.
```

Comment:
This sentence is repeated

Reply:
You can find the change here: https://github.com/cerner/fhir.cerner.com/commit/488b91d2667011cecf4dd1bc150914e97c7dff73